orttraining/orttraining/python/training/ortmodule/_training_manager.py

```diff
@@ -451,9 +449,11 @@ def _create_execution_agent(self):
             )
 
             self._runtime_inspector.memory_ob.find_memory_optimization_opportunity(
-                execution_agent, self._runtime_options.memory_optimizer_config, self._runtime_options.probe_level
+                execution_agent, self._runtime_options
             )
 
+            self._logger.debug("Completed find_memory_optimization_opportunity.")
```

Comment:
Is this necessary? Can this be moved inside find_memory_optimization_opportunity?